README.md

```diff
@@ -450,6 +452,47 @@ else {
 }
 ```
 
+## `OIDC` token.parseAndStoreTokensFromUrl(tokenStorageKeys)
+
+Parse and **store** the access or ID Tokens from the url after a successful authentication redirect. This is used when `authParams.display = 'page'`.
+
+* `tokenStorageKeys` - Override the default storage naming conventions for the access and/or ID tokens. Defaults to `accessToken` and `idToken`.
+
+```javascript
+var signIn = new OktaSignIn({
+  baseUrl: 'https://{yourOktaDomain}',
+  clientId: '{{myClientId}}',
+  redirectUri: '{{redirectUri configured in OIDC app}}',
+  authParams: {
+    responseType: 'id_token',
+    // `display: page` will initiate the OAuth2 page redirect flow
+    display: 'page'
+  }
+});
+
+// The user has just landed on our login form, and has not yet authenticated
+if (!signIn.token.hasTokensInUrl()) {
+  signIn.renderEl({el: '#osw-container'});
+}
+
+// The user has redirected back after authenticating and has their access and/or
+// ID Token in the URL.
+else {
+  signIn.token.parseAndStoreTokensFromUrl()
+  .then(function() {
+    const idToken = signIn.tokenManager.get('idToken');
```

Comment:
Would making idToken in Caps make it look like it's constant?

signIn.tokenManager.get('ID_TOKEN');

This way even the next part where we add alias can look more explicit:

signIn.token.parseAndStoreTokensFromUrl({ 'ID_TOKEN': 'my-id-token' })

Reply:
Great idea! When I was thinking about this originally, I wanted to stick with the camelcase convention, but this is MUCH more explicit. 👍